src/modules/ChannelList/dux/getNextChannel.ts

```diff
@@ -0,0 +1,33 @@
+// See the conditions of this function here https://github.com/sendbird/sendbird-uikit-react/pull/777
+import { GroupChannel } from '@sendbird/chat/groupChannel';
+
+type CurrentChannelType = GroupChannel | null;
+
+export interface GetNextChannelParams {
+  channel: GroupChannel;
+  currentChannel: CurrentChannelType;
+  allChannels: GroupChannel[];
+  disableAutoSelect: boolean;
+}
+
+/**
+ * NOTICE: Use this function IF the current channel is removed from allChannels.
+ * This function will give you the next currentChannel value.
+ */
+export const getNextChannel = ({
+  channel,
+  currentChannel,
+  allChannels,
+  disableAutoSelect,
+}: GetNextChannelParams): CurrentChannelType => {
+  let nextChannel: CurrentChannelType = null;
```

Comment:
Do you mean? 
```suggestion
  let nextChannel: CurrentChannel = null;
```

Reply:
CurrentChannel is not defined but
CurrentChannelType is defined above